frontend/pages/queries/QueryPage/components/QueryForm/QueryForm.tsx

```diff
@@ -424,7 +424,7 @@ const QueryForm = ({
           name="query editor"
           label="Query"
           wrapperClassName={`${baseClass}__text-editor-wrapper`}
-          readOnly={!isObserverPlus || !isAnyTeamObserverPlus}
+          readOnly={!isObserverPlus && !isAnyTeamObserverPlus}
```

Comment:
Is this gonna make it so that on both the new live query page AND the edit saved query page, the field will be editable?

Reply:
I think we don't want that without more thought. A user should not be able to edit a saved query (but not update it), run it live, and then get an audit event that would look like they ran the saved query.

Reply:
That is indeed what this gives us. Looking into separating this logic.

Reply:
Updated code should solve this